pipeline_reviews/extract.py

```diff
@@ -68,13 +73,7 @@ def get_all_reviews(game_ids: list[int]) -> None:
                     if not cursor in cursor_list:
                         cursor_list.append(cursor)
                     all_reviews.extend(page_reviews)
-    make_csv_files(all_reviews)
-
-
-def make_csv_files(all_reviews: list[dict]) -> None:
```

Comment:
Clarity: `make_csv_files` sounds like it makes CSV files (which is true) but it also returns a `DataFrame` which is confusing. I would consider two options:

- Split the function into two, one to write to a CSV and one to return a `DataFrame` from the input data
- Rename the function e.g. `write_csv_and_get_dataframe`

The type hint for this function also needs to be updated accordingly

Reply:
So, now this function was fully removed (which is why it is in red)